day5/README.md

```diff
@@ -0,0 +1,29 @@
+Question of the day: https://leetcode.com/problems/path-sum/#/description
+
+"Given a binary tree and a sum, determine if the tree
+has a root-to-leaf path such that adding up all the
+values along the path equals the given sum."
+
+## Ideas ##
+
+The tree isn't organized, i.e. it's not a binary search
+tree. That means I'll have to do some kind of extensive
+searching, and since I want a path, DFS should work.
```

Comment:
for a challenge, try this with BFS as well. or change the problem description from "root-to-leaf path" to "any path starting from the root" or "any path traveling down the tree" or "any path at all"